src/Tools/dotnet-trace/CommandLine/Commands/CollectCommand.cs

```diff
@@ -61,6 +59,14 @@ private static async Task<int> Collect(CancellationToken ct, IConsole console, i
             bool cancelOnCtrlC = true;
             bool printStatusOverTime = true;
             int ret = ReturnCode.Ok;
+            if (showchildio)
```

Comment:
One last nit picky detail, this would be a lot simpler and easier to read if it was just `IsQuiet = shochildio;`